src/object.js

```diff
@@ -334,6 +334,14 @@ Sk.builtin.object.prototype["__ge__"] = function (self, other) {
     return self.ob$ge(other);
 };
 
+Sk.builtin.object.prototype.__get__ = function __get__(self, instance, owner) {
+    Sk.builtin.pyCheckArgs("__get__", arguments, 1, 2, false, true);
+
+    var l_owner = !owner ? instance.ob$type : owner;
+
+    return self.tp$descr_get(instance, l_owner);
```

Comment:
In CPython, `object` does not automatically have a `__get__` method.

I appreciate we'd prefer not to rewrite a generic definition of `__get__` for every type that implements `tp$descr_get()`, but adding a method to `object` that shouldn't be there isn't the answer.

Perhaps we should have a function that uses the `dunderToSkulpt` mapping (from `type.js`) in reverse - ie, given a native implementation of a slot (like `tp$descr_get`), it generates a Python-callable version of that function (in this case `__get__`)? Something like `makeDunderFunction(Sk.builtin.func.prototype, "__get__")`?

Reply:
Ohh right yeah of course there isn't, I'll have to rethink this 😁.